pandas/tests/indexes/multi/test_get_set.py

```diff
@@ -27,6 +27,11 @@ def test_get_level_number_integer(idx):
         idx._get_level_number("fourth")
 
 
+def test_get_dtypes(idx):
```

Comment:
can you add the issue number as a comment

i think idx is too simple, can you construct a MI like in the OP and test it here